amazon-s3/pom.xml

```diff
@@ -28,7 +28,7 @@
         <dependency>
             <groupId>software.amazon.awssdk</groupId>
             <artifactId>s3</artifactId>
-            <version>2.18.35</version>
+            <version>2.19.8</version>
```

Comment:
`2.19.12` is already available. Is there any concern for using the most recent version?